gantry/firmware/axis_hardware_config.h

```diff
@@ -1,11 +1,14 @@
+#pragma once
+
+#include "stdint.h"
 #include "platform_specific_hal_conf.h"
 
-typedef struct {
+typedef struct PinType {
```

Comment:
i'm perfectly fine with c anonymous structs and enums, since the `struct PinType` is never explicitly used - any reason for this change?

Reply:
No reason really. I can keep it anonymous.